src/views/Settings/CurrencyTypeForm/index.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+  section
+    form
+      input(type="checkbox" value="Bitcoin" id="bitcoin" v-model="chosenCryptoCurrency")
+      label(for="bitcoin") Bitcoin
+      input(type="checkbox" value="Bitcash" id="bitcash" v-model="chosenCryptoCurrency")
+      label(for="bitcash") Bitcash
+      input(type="checkbox" id="etherium" value="Etherium" v-model="chosenCryptoCurrency")
+      label(for="etherium") Etherium
+      input(type="checkbox" id="littleCoin" value="LittleCoin" v-model="chosenCryptoCurrency")
+      label(for="littleCoin") LittleCoin
+      select(v-model="selectedCurrency")
+        option(selected="true" value="USD") USD
```

Comment:
Tutaj lepiej było by zrobić `computed` property np. `currencies() { }`, które zwraca tablicę obiektów (`[{ value: "PLN", selected: false }, { value: "USD", selected: true }, ... ]`) i iterować po tej tablicy `option(v-for="currency in currencies", :value="currency.value", :selected="currency.selected") {{  currency.value }}`

Reply:
tak myślałem ale chciałem na początek za hardkodować to już poprawiam